lib/core/elastic_connector_actions.rb

```diff
@@ -36,6 +37,38 @@ def get_connector(connector_id)
         client.get(:index => CONNECTORS_INDEX, :id => connector_id, :ignore => 404).with_indifferent_access
       end
 
+      def native_connectors
+        result = []
+        offset = 0
+        loop do
+          response = client.search(
+            :index => CONNECTORS_INDEX,
+            :ignore => 404,
+            :body => {
+              :size => DEFAULT_PAGE_SIZE,
+              :from => offset,
+              :query => {
+                :term => { :is_native => true }
+              },
+              :sort => ['name']
```

Comment:
I tried it at first, but I was having this exception:

`"Fielddata access on the _id field is disallowed"`

I think we might be missing something in our mappings for this index, but I opted for sorting on something else, rather than blowing out the scope of this task.